node-launchpad/src/components/resource_allocation.rs

```diff
@@ -21,7 +21,8 @@ use crate::{
 use super::{utils::centered_rect_fixed, Component};
 
 pub const GB_PER_NODE: usize = 5;
-pub const GB: usize = 1024 * 1024 * 1024;
+pub const MB: usize = 1000 * 1000;
+pub const GB: usize = MB * 1000;
```

Comment:
Just a nitpick, but I think we're using `MiB`/`GiB` (mebibyte/gibibyte) mostly in our apps (1024/binary based), lining up with chunks which are 512KiB(?).

Reply:
I used `MiB` earlier but here this is for fs storage and I crosschecked the space used by a folder with the ones from macos -> right click -> info and they did not match. So I assume we should be using `MB/GB` for fs?